### PR TITLE
update openapi version to v1.20.4

### PR DESCRIPTION
**Pull request by natasha41575**
ALLOW_MODULE_SPAN

### CODE REVIEW

File: kyaml/openapi/kubernetesapi/openapiinfo.go [hunk 1]
```diff
@@ -6,21 +6,13 @@
...
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1184"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1186"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1188"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1190"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1191"
+	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1204"
 )
 
-const Info = "{title:Kubernetes,version:v1.18.4}\n{title:Kubernetes,version:v1.18.6}\n{title:Kubernetes,version:v1.18.8}\n{title:Kubernetes,version:v1.19.0}\n{title:Kubernetes,version:v1.19.1}"
+const Info = "{title:Kubernetes,version:v1.19.0}"
```
Comment (liggitt) [inline kyaml/openapi/kubernetesapi/openapiinfo.go:12]: should this be 1.20.4?

Comment (natasha41575) [reply kyaml/openapi/kubernetesapi/openapiinfo.go:12]: ah you're right, an error in the script. Will fix, thanks for the catch

**Comment by monopole** [comment]
so in the kyaml dir, `golangci-lint ./...` is complaining `golang.org/x/net/idna: no go files to analyze`
FWIW
```
~/gopath/src/sigs.k8s.io/kustomize/kyaml $ go mod why  golang.org/x/net/idna
# golang.org/x/net/idna
sigs.k8s.io/kustomize/kyaml/fieldmeta
github.com/go-openapi/spec
github.com/go-openapi/jsonreference
github.com/PuerkitoBio/purell  <-- a URL normalizer/sanitizer
golang.org/x/net/idna
```

According to https://github.com/golangci/golangci-lint/issues/825, the error message is poor,.

Maybe try https://github.com/golangci/golangci-lint/issues/825#issuecomment-666038142

**Comment by monopole** [comment]
And by try,  i mean in the kyaml dir, rm go.sum, clear the mod cache, run go tidy, commit and push

**Review by monopole — APPROVED** [review]
/lgtm

thanks!

**Pull request merged**